src/pages/Trades/TradesDetailsModal.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Heading, Icon, Modal, Text } from '@unique-nft/ui-kit';
+import styled from 'styled-components';
+import { TokenComponent } from './TokenComponent/TokenComponent';
+import { AdditionalDark, BlueGrey600 } from 'styles/colors';
+import { timestampTableFormat } from 'utils/timestampUtils';
+import { AddressComponent } from './AddressComponent/AddressComponent';
+import Winner from 'static/icons/winner.svg';
+import { Table } from 'components/Table';
+import { useTradeDetails } from 'api/restApi/trades/tradeDetails';
+import { Bid, Trade } from 'api/restApi/trades/types';
+import { formatKusamaBalance } from 'utils/textUtils';
+
+export interface IConfirmModalProps {
+  trade: Trade | null
+  onCancel: () => void
+}
+const tokenSymbol = 'KSM';
+
+const columns = [
+  {
+    title: 'Bid',
+    width: '33.33%',
+    render: (value: string) => <Text color={AdditionalDark}>{`${formatKusamaBalance(value)} ${tokenSymbol}`}</Text>,
+    field: 'balance'
+  },
+  {
+    title: 'Time',
+    width: '33.33%',
+    isSortable: true,
+    render: (time: number) => <TimeCell><Text color={BlueGrey600}>{timestampTableFormat(new Date(time).valueOf())}</Text></TimeCell>,
+    field: 'updatedAt'
+  },
+  {
+    title: 'Bidder',
+    width: '33.33%',
+    render: (data: string, row: Bid) => <BidderCell><AddressComponent text={data} />{row.isWinner && <Icon size={32} file={Winner}/>}</BidderCell>,
+    field: 'bidderAddress'
+  }
+];
+
+const TokenTradesDetailsModal: FC<IConfirmModalProps> = ({ trade, onCancel }) => {
+  const [filteredBids, setFilteredBids] = useState<Bid[]>();
+  const { tradeDetails, fetch, isFetching } = useTradeDetails();
+  useEffect(() => {
+    if (trade) {
+      fetch(trade.offerId);
+    }
+  }, [fetch, trade]);
+
+  useEffect(() => {
+    if (tradeDetails?.bids) {
```

Comment:
if you open a trade with bids and afterwards open a trade without any - u will see previous trade bids due to that "if"

Reply:
it's kinda impossible since we can't have a finished trade without a single bid